tests/test_constraints_polars.py

```diff
@@ -27,13 +30,16 @@ def _lazyframe_from_product(parameters):
     return res
 
 
+@pytest.mark.skipif(
```

Comment:
I think it makes more sense to go with a module-wise `pytestmark = pytest.mark.skipif(...)` (see import tests) instead of tagging every single test in this file

Reply:
ok changed
I also found the `pytest.skipimport` which seems very suitable here. But if that triggers the test (and neither of the subtests) shows up in the console which I find a bit bad